src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -350,3 +383,11 @@
     type: group
     short-summary: Manage Azure Active Directory users and user authentication.
 """
+helps['ad signed-in-user'] = """
```

Comment:
Can these be folded into the existing `az ad user` group?  For example `az ad user show` requires `--upn-or-object-id`, but what if, if I supplied nothing, by default to shows the current user (just like `az account show`)?  `list-owned-objects` might need to be it's own command, but it seems less awkward than having a `sign-in-user` group.

Reply:
I thought about the same thing, and felt can go either way
Let me cross check whether or not we have other signed-in-user's operations not yet captured in swagger. If none, i will pick up your suggestions.

Reply:
Cross checked the [graph api](https://msdn.microsoft.com/Library/Azure/Ad/Graph/api/api-catalog), looks like there are more operations for singed-in-user, like get/assign manager, etc.
Let us keep it as a separated group